src/Tracker/utils/index.js

```diff
@@ -0,0 +1,45 @@
+export const sendAnalytics = (payload) => {
+  const request = new XMLHttpRequest()
+  request.open('POST', 'https://analytics-sdk-dev.onfido.com/v1/import')
+  request.setRequestHeader('Content-Type', "application/json")
+  request.setRequestHeader('Authorization', "Basic M002Z3tQWURwWTlUanlDKDNRTHIqTDZOQ1hVY2FLZUU6")
+
+  request.onload = () => {
+    if (request.status === 200 || request.status === 201) {
+      console.log(request.response)
+    }
+    else {
+      console.log(request)
+    }
+  }
+  request.onerror = () => console.log(request)
+
+  request.send(payload)
+}
+
+export const formatAnalytics = (woopra, eventName, properties) =>
```

Comment:
aren't you missing `referer` like this? And maybe other important values?

Reply:
Yes, there is some info missing. This is just a basic "working" example (where the service returns 200). I will add the missing info now. Btw, I haven't checked yet if woopra is actually receiving this data from the service cause there were some issues on the woopra dashboard last week.

Reply:
> This is just a basic "working" example (where the service returns 200).

Please *do not* trust the 200 as everything going well. Really. Please double check in woopra. Small note, the current backend will use the mobile projects, not the JS..... that is something we also need to solve.